src/main/java/fr/dynamx/api/physics/player/DynamXPhysicsWorldBlacklistApi.java

```diff
@@ -0,0 +1,18 @@
+package fr.dynamx.api.physics.player;
+
+import net.minecraft.entity.player.EntityPlayer;
+
+public class DynamXPhysicsWorldBlacklistApi {
+
+    private static IPhysicsWorldBlacklist blackListCondition;
```

Comment:
Doing 
`private static IPhysicsWorldBlacklist blackListCondition = player -> false;` will be better than using null as default value.